src/Controller/AdminController.php

```diff
@@ -105,8 +105,8 @@ public function getOutputConfigsAction(Request $request)
                     'id' => $def->getId(),
                     'classname' => $translator->trans($class->getName(), [], 'admin'),
                     'channel' => $translator->trans($channel, [], 'admin'),
-                    'object_id' => $def->getO_Id(),
-                    'is_inherited' => $def->getO_Id() != $objectId
+                    'object_id' => $def->getId(),
```

Comment:
there's something that doesn't look right here, `object_id` would be the same as `id` on [line 105](https://github.com/pimcore/output-data-config-toolkit/pull/70/files#diff-15f7f071121bfa3ffa4bf6f5b31db091f12a2ce65acae1642ab42c45e0c9eef7R105)
The right method is [getObjectId()](https://github.com/pimcore/output-data-config-toolkit/pull/70/files#diff-4026997d615e34ef8e443eecd0b6cf4c96eaf729587e08872658afe9a26ee96dR148)

Reply:
Proposing the right fixes in https://github.com/pimcore/output-data-config-toolkit/pull/76